types/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// TypeScript Version: 3.0
+// TypeScript Version: 3.4
```

Comment:
This also works in TypeScript 3.0, but Variadic parameter length checking wasn't added until 3.4

So the test on line 
```ts
// $ExpectError
processor.use(implicitlyTypedPlugin, typedSetting, settings)
```

doesn't get it's expected error

```
ERROR: 102:1  expect  TypeScript@3.1: Expected an error on this line, but found none.
```

this could be reverted back to `3.0` and a note added over the test.